ytmusicapi/mixins/playlists.py

```diff
@@ -232,6 +232,10 @@ def add_playlist_items(self,
                 action['dedupeOption'] = 'DEDUPE_OPTION_SKIP'
             body['actions'].append(action)
 
+        # add an empty ACTION_ADD_VIDEO because otherwise YTM doesn't return the dict that maps videoIds to their new setVideoIds
+        if not videoIds:
```

Comment:
I assume this is needed only if a source playlist is used?

Reply:
yep